packages/node_modules/proxy-state-tree/src/Proxyfier.ts

```diff
@@ -78,9 +78,10 @@ export class Proxifier {
     if (this.tree.master.options.devmode && !this.tree.canMutate()) {
       throw new Error(
         `proxy-state-tree - You are mutating the path "${path}", but it is not allowed. The following could have happened:
-        
+
         - You are passing state to a 3rd party tool trying to manipulate the state
         - You are running asynchronous code and forgot to "await" its execution
+        - You are attempting to mutate state outside of an action
```

Comment:
This is awesome, though `proxy-state-tree` does not really have actions. So if you could change the text to something like: "You are attempting to mutate state where it is not allowed"... or something like that :)